documentation/en/Chapter_7.md

```diff
@@ -0,0 +1,155 @@
+# 7. Selection of the Operating System.
+
+The criteria for choosing an operating system (OS) in this case are similar to those criteria of a sensor network.
+
+Our network device must have certain special features, [4,5,6] including:
+- Low cost.
+- Low energy consumption.
+- It can run on battery.
+To achieve the objective, an efficient network protocol such as 6LowPAN should be used to reduce transmission time and save energy.
```

Comment:
```suggestion
To achieve the goal, an efficient network protocol such as 6LowPAN should be used to reduce transmission time and save energy.
```

Reply:
modificado, thanks

Reply:
Resolved in 1fd01e9b40fc61d8e297bf860f107709cfeccd52